### PR TITLE
feat: due, follow up, and completion dates are shown as relative dates

### PR DESCRIPTION
**Pull request by douglasbouttell-camunda**
## Description

Clone of https://github.com/camunda/tasklist/pull/4847

Use relative date logic to show due date, overdue date, completion date and follow-up date.

## Related issues

closes https://github.com/camunda/tasklist/issues/4662

### CODE REVIEW

**Review by vsgoulart — APPROVED** [review]

File: tasklist/client/e2e/pages/TaskPanelPage.ts [hunk 1]
```diff
@@ -49,12 +49,8 @@ class TaskPanelPage {
     }
   }
 
-  task(name: string) {
-    return this.availableTasks.getByLabel(name).nth(0);
-  }
-
   async openTask(name: string) {
-    await this.task(name).click();
+    await this.availableTasks.getByText(name, {exact: true}).nth(0).click();
```
Comment (vsgoulart) [inline tasklist/client/e2e/pages/TaskPanelPage.ts:53]: @douglasbouttell-camunda why was this necessary?

Comment (douglasbouttell-camunda) [reply tasklist/client/e2e/pages/TaskPanelPage.ts:57]: Trying to fix/debug the end to end tests

Comment (vsgoulart) [reply tasklist/client/e2e/pages/TaskPanelPage.ts:57]: E2E tests are flaky in the CI right now
I plan on checking them, but if you have spare time please go ahead

It's better to download the report from the CI to analyse them
